### DISCUSSION TITLE
Configuring Sunbird RC for a State Program

### DISCUSSION THREAD

**Discussion by karsood** [Sunbird-RC/community#151] [General]
Hi @parthlawate,
I work on the program team at EkStep and wanted to get a broad understanding of the steps involved in enabling a state department to create a registry.
The context of the program is that the state is planning to roll out devices to each student. These devices can be mapped to students so that usage can be monitored.

I would like to further engage on this. Requesting for a connect via email/call.

Regards,
Karan

**Comment by parthlawate** [comment]
Hi sure Karan. Will Monday work ? I'm travelling this week

**Reply by karsood** [reply]
Sure Parth, that works. Let me email you to take this forward.

**Comment by ghost** [comment]
I am also interested in getting a broad understanding of the steps involved in enabling a state department to create a registry. My context is 8 universities, 15 TEVET colleges
